kubernetes/yamls/susi-server/susi-deployment.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Deployment of the SUSI Server
+#
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: susi-server
+  namespace: web
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: susi-server
+    spec:
+      nodeSelector:
+        server: primary
+      containers:
+      - name: susi-server
+        image: chiragw15/susi_server2:latest
+        volumeMounts:
+          - mountPath: /data
```

Comment:
@chiragw15: [This](https://github.com/fossasia/susi_server/blob/development/src/ai/susi/SusiServer.java#L157) tries to get relative path `data`. I guess it should be `/susi_server/data`.

Have you tried opening a shell in running node and checking how files are arranged? It would clarify things.